doc/Security.xml

```diff
@@ -947,13 +995,311 @@ Added certificate-based client authentication</revremark>
         </orderedlist>
       </section>
     </section>
+    <section xml:id="_Ref395181339">
+      <title>JWT-based client authorization</title>
+      <para>Unlike with HTTP Digest and WS-UsernameToken authentication, this specification defines how to associate clients with the different User Levels as defined in the ONVIF Core Specification. Devices are not expected to have the user credentials stored in their memory, instead they will verify that the user has been correctly authenticated by a trusted service based on OpenID Connect and authorize accessing different functions, based on the claims presented in the supplied JWT. JWTs can be presented has headers in case of HTTPS or RTSP, or they can be embedded in SOAP messages, in the form of binary security tokens. In case JWTs are embedded in binary security tokens, their content are the base64-encoded JWTs according to [RFC 7519].</para>
+      <para>JWTs consists of three elements:</para>
+      <para>
+        <itemizedlist>
+          <listitem>
+            <para>JOSE header</para>
+          </listitem>
+          <listitem>
+            <para>Payload</para>
+          </listitem>
+          <listitem>
+            <para>Signature</para>
+          </listitem>
+        </itemizedlist>
+      </para>
+      <para>The JOSE header shall declare that the encoded object is a JSON Web Token by setting the <emphasis>typ</emphasis> claim to <emphasis>JWT</emphasis>, and the JWT is a JWS that is signed using the asymmetric algorithm identified by the <emphasis>alg</emphasis> claim with a value defined in RFC7518. Since the keystore does not support symmetric encryption, symmetric keyed hashing algorithms shall not be used.</para>
```

Comment:
symmetric keyed hashing algorithms shall not be used => out of scope? ONVIF normally don't prohibit  extensions.